SpecialManageWiki.php

```diff
@@ -109,6 +109,14 @@ function showWikiForm( $wiki ) {
 				'options' => $options,
 				'name' => 'mwLanguage',
 			),
+			'logo' => array(
+				'label-message' => 'managewiki-label-logo-url',
+				'type' => 'text',
+				'size' => 20,
+				'default' => 'https://static.miraheze.org/metawiki/3/35/Miraheze_Logo.svg',
```

Comment:
This is definitely possible. We should be able to use a validation callback to ensure that only static.miraheze.org files are set as the logo. 

See here: https://www.mediawiki.org/wiki/HTMLForm/tutorial2#Adding_Validation